test/shared/src/main/scala/zio/test/ZSpecRunner.scala

```diff
@@ -28,15 +28,55 @@ object ZSpecRunner {
    * Runs tests in parallel, up to the specified limit.
    */
   def parallel(n: Int): ZSpecRunner = new ZSpecRunner {
-    val _ = n
-
-    def run[R, E, L](spec: ZSpec[R, E, L]): UIO[ExecutedSpec[R, E, L]] = ???
+    def run[E, L](spec: ZSpec[Any, E, L]): UIO[ExecutedSpec[Any, E, L]] =
```

Comment:
Can you change this to be polymorphic in `R`? You just need to change the return type from `UIO` to `RIO` (sorry, my bad).

Otherwise, looks great!